spyder/plugins/editor/widgets/tests/test_goto.py

```diff
@@ -0,0 +1,66 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright © Spyder Project Contributors
+# Licensed under the terms of the MIT License
+#
+"""Tests for editor URI and mailto go to handling."""
+
+# Standard library imports
+import os
+import tempfile
+
+# Third party imports
+from qtpy.QtCore import Qt, QPoint
+from qtpy.QtGui import QTextCursor
+import pytest
+
+# Constants
+TEST_FOLDER = os.path.abspath(os.path.dirname(__file__))
+_, TEMPFILE_PATH = tempfile.mkstemp()
+TEST_FILE_TEXT = '"file://{}"\n'.format(TEMPFILE_PATH)
+
+
+@pytest.mark.slow
+@pytest.mark.second
+@pytest.mark.parametrize('params', [
+            # Parameter, Expected output
+            # --------------------------
+            # Urls
+            ('" https://google.com"\n', 'https://google.com'),  # String
+            ('# https://google.com"\n', 'https://google.com'),  # Comment
+            # Files that exist
+            (TEST_FILE_TEXT, TEMPFILE_PATH),  # File without spaces
+            # Mail to
+            ('" mailto:goanpeca@gmail.com"\n', 'mailto:goanpeca@gmail.com'),
+            ('# mailto:goanpeca@gmail.com\n', 'mailto:goanpeca@gmail.com'),
+        ]
+    )
+def test_goto_uri(qtbot, lsp_codeeditor, params):
```

Comment:
There's no need to use `lsp_codeeditor` because this doesn't depend on the LSP facilities at all. Since that's costly and prone to generate conflicts with other tests, we should only use it when it's really necessary.

So please use a fixture that creates a CodeEditor without LSP activated instead.

Reply:
There's a `code_editor_bot` or a similar fixture used in a lot of tests in `spyder/plugins/editor/widgets/tests`. Please move it to `spyder/plugins/editor/widgets/tests/fixtures.py` and use it for all tests that need it (including this one).